SDKGenerator.njsproj

```diff
@@ -461,142 +461,264 @@
-    <Content Include="targets\unity-v2\source\Shared\Ionic.Zlib\ZTree.cs.meta" />
-    <Content Include="targets\unity-v2\source\Shared\Models.meta" />
-    <Content Include="targets\unity-v2\source\Shared\Models\Editor\MakeSharedSettingsObj.cs" />
-    <Content Include="targets\unity-v2\source\Shared\Models\PlayFabSharedSettings.cs.ejs" />
-    <Content Include="targets\unity-v2\source\Shared\Models\PlayFabSharedSettings.cs.meta" />
-    <Content Include="targets\unity-v2\source\Shared\Models\SharedModels.cs" />
-    <Content Include="targets\unity-v2\source\Shared\Models\SharedModels.cs.meta" />
-    <Content Include="targets\unity-v2\source\Shared\Public.meta" />
-    <Content Include="targets\unity-v2\source\Shared\Public\IPlayFabPlugin.cs" />
-    <Content Include="targets\unity-v2\source\Shared\Public\IPlayFabPlugin.cs.meta" />
-    <Content Include="targets\unity-v2\source\Shared\Public\ISerializerPlugin.cs" />
-    <Content Include="targets\unity-v2\source\Shared\Public\ISerializerPlugin.cs.meta" />
-    <Content Include="targets\unity-v2\source\Shared\Public\ITransportPlugin.cs" />
-    <Content Include="targets\unity-v2\source\Shared\Public\ITransportPlugin.cs.meta" />
-    <Content Include="targets\unity-v2\source\Shared\Public\PlayFabApiSettings.cs.ejs" />
-    <Content Include="targets\unity-v2\source\Shared\Public\PlayFabAuthenticationContext.cs" />
-    <Content Include="targets\unity-v2\source\Shared\Public\PlayFabDataGatherer.cs" />
-    <Content Include="targets\unity-v2\source\Shared\Public\PlayFabDataGatherer.cs.meta" />
-    <Content Include="targets\unity-v2\source\Shared\Public\PlayFabLogger.cs" />
-    <Content Include="targets\unity-v2\source\Shared\Public\PlayFabLogger.cs.meta" />
-    <Content Include="targets\unity-v2\source\Shared\Public\PlayFabSettings.cs.ejs" />
-    <Content Include="targets\unity-v2\source\Shared\Public\PlayFabSettings.cs.meta" />
-    <Content Include="targets\unity-v2\source\Shared\Public\PluginContract.cs" />
-    <Content Include="targets\unity-v2\source\Shared\Public\PluginContract.cs.meta" />
-    <Content Include="targets\unity-v2\source\Shared\Public\PluginContractKey.cs" />
-    <Content Include="targets\unity-v2\source\Shared\Public\PluginManager.cs" />
-    <Content Include="targets\unity-v2\source\Shared\Public\PluginManager.cs.meta" />
-    <Content Include="targets\unity-v2\source\Shared\Public\Resources.meta" />
-    <Content Include="targets\unity-v2\source\Shared\Public\Resources\PlayFabSharedSettings.asset" />
-    <Content Include="targets\unity-v2\source\Shared\Public\Resources\PlayFabSharedSettings.asset.meta" />
+    <Content Include="targets\unity-v2\source\ExampleTestProject\Assets\PlayFabEditorExtensions\Editor.meta" />
+    <Content Include="targets\unity-v2\source\ExampleTestProject\Assets\PlayFabEditorExtensions\Editor\PlayFabEditor.cs" />
+    <Content Include="targets\unity-v2\source\ExampleTestProject\Assets\PlayFabEditorExtensions\Editor\PlayFabEditor.cs.meta" />
```

Comment:
Can we leave the .meta files out of the SDKGenerator? These will prompt Unity to upgrade our example test project every time we open it (There's too many versions of Unity floating around to make this consistent). The meta files will be generated when they first load into Unity automatically. I think we should leverage that ability.